cmd/snap/cmd_help.go

```diff
@@ -204,7 +204,7 @@ var helpCategories = []helpCategory{
 	{
 		Label:       i18n.G("Basics"),
 		Description: i18n.G("basic snap management"),
-		Commands:    []string{"find", "info", "install", "remove", "list"},
+		Commands:    []string{"find", "info", "install", "remove", "list", "components"},
```

Comment:
nitpick: not sure this is a basic command. I guess there's no section it clearly belongs to